charts/gha-runner-scale-set-controller/values.yaml

```diff
@@ -102,3 +102,14 @@ flags:
   ##   This can lead to a longer time to apply the change but it will ensure
   ##   that you don't have any overprovisioning of runners.
   updateStrategy: "immediate"
+
+# # If metrics object is not provided, following flags won't be applied:
+# # --metrics-addr, --listener-metrics-addr, --listener-metrics-endpoint
+# # Keep in mind, metrics are exposed by default (flags have default values)
+# # specifying the default port and default endpoint.
+# #
+# # To turn off metrics, specify empty strings for controllerAddr and listenerAddr
```

Comment:
Maybe we should remove this comment since they're disabled by default.

Reply:
I have to update the comment above as well